### PR TITLE
feat: intro, en version

### PR DESCRIPTION
**Pull request by ldxdl**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [ ] Y 

#### 2. What is the scope of this PR (e.g. component or file name):
https://kusionstack.io/docs/user_docs/intro/kusion-intro

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/user_docs/intro/kusion-intro.md [hunk 1]
```diff
@@ -4,90 +4,92 @@
...
+在蚂蚁我们经历了以上这些问题，我们构建了基于[裸金属](https://en.wikipedia.org/wiki/Bare-metal_server)的超大规模 Kubernetes 集群，基于 [MOSN](https://github.com/mosn/mosn) sidecar 的规模化 mesh 体系，同时也大量依赖中间件等 PaaS 服务及部分 IaaS 云服务。在运维侧，我们做了很多尝试来适应规模、复杂性和快速变化，如加入云原生元素改良经典运维平台，如采纳 Kubernetes 社区原生的 YAML-based DevOps 运维方案，但都无法满足内部以应用为中心的高效率、低感知，并兼顾开放性、灵活、可扩展、可移植的规模化交付运维需求。
 
 
 ![](/img/docs/user_docs/intro/kusion-stack-1.png)
 
 
-KusionStack 通过 engineering（工程化）的理念和技术栈融合平台方，研发者和 [SRE](https://en.wikipedia.org/wiki/Site_reliability_engineering)，让平台方更简单灵活的开放平台能力，让应用研发者以应用为中心工作，同时降低研发者在参与运维过程中对基础设施的认知负担，同时又赋予研发者充分的灵活性。总的来说，KusionStack 致力于构建**以应用为中心的抽象界面**，**一致的管理工具及自动化支持**和**更简单的使用体验和工作流程**，并希望趋近这样的理性状态：
+KusionStack 通过 engineering（工程化）的理念融合平台方，研发者和 [SRE](https://en.wikipedia.org/wiki/Site_reliability_engineering)，通过以 “modeling on platforms, compile to cloud” 为目标的一站式技术栈，让平台方简单灵活的开放平台能力，让应用研发者以应用为中心设计交付运维抽象，降低研发者在参与运维过程中对基础设施的认知负担，同时又赋予研发者充分的灵活性。总的来说，KusionStack 与其他代码化技术一样提供了**版本化**、**可重用**、**可重复**的便利，同时 KusionStack 致力于构建**以应用为中心的抽象界面**，**一致的管理工具及自动化支持**和**更简单的端到端使用体验和工作流程**，并希望趋近这样的理性状态：
```
Comment (Peefy) [inline docs/user_docs/intro/kusion-intro.md:30]: 个人感觉 ”可重复“ 是不是改成 “可扩展” 或者 “可测试” 好一些，【代码重复】似乎不是一个正向词汇

File: docs/user_docs/intro/kusion-intro-en.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+# Introduction to KusionStack
+
+## What is KusionStack?
+
+**KusionStack is an open-source cloud-native programmable technology stack!**
+
+KusionStack is a highly flexible programmable technology stack to enable unified application delivery and operation, inspired by the word Fusion, which aims to help enterprises build an application-centric configuration management plane and DevOps ecology.
```
Comment (SparkYuan) [inline docs/user_docs/intro/kusion-intro-en.md:7]: 没查到DevOps ecology这种说法，只查到了DevOps ecosystem这种表述

File: docs/user_docs/intro/kusion-intro-en.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+1. Fusion of hybrid scenarios of **private cloud**, **hybrid cloud**, and **multi-cloud** 
+2. Fusion of hybrid platform technologies of **cloud-native** and a variety of other platform technologies
```
Comment (SparkYuan) [inline docs/user_docs/intro/kusion-intro-en.md:10]: platform technologies所谓的平台技术有点抽象，这里和cloud-native对应的是指tf这种传统资源吗

File: docs/user_docs/intro/kusion-intro.md [hunk 4]
```diff
@@ -4,90 +4,92 @@
...
+KusionStack 源于蚂蚁集团内部的规模化工程化运维实践，已广泛应用在蚂蚁 SaaS 应用多云交付运维，计算及数据基础设施交付，建站运维，数据库运维等多个业务领域，助力蚂蚁完成从传统运维管理模式到协同开放的可编程 DevOps 运维模式的转型。
 
 
-## 为什么使用 Kusion？
+## 为什么使用 KusionStack？
 
-以 Kubernetes 为代表的云原生技术正成为现代企业及云厂商的一方技术，并快速形成面向全球的社区生态。Kubernetes API 支持面向终态、数据化（as Data）的定义方式，声明式、版本化、面向资源的可扩展 API，可终止、可预测的服务端模拟验证机制，这些优秀的设计使其天然具有一致的接入方式，完善的资源定义和扩展方式，以及客户端友好的验证、执行机制，原生具备了传统三方 API 接入层技术产品的典型能力。但从另一个角度看，现代应用也将同时长期依赖未云原生化的 PaaS、IaaS 层服务，这使得企业内应用交付运维依赖分散割裂的技术和平台，复杂混乱的工作界面和流程。此外，面向大量的非专家型应用研发者，Kubernetes 自身的技术复杂性和大量面向资源的使用界面制约了在企业内部以开放、民主方式建设更为广泛的 DevOps 生态。最后，平台侧缺少有效手段让大量平台能力简单、可控地开放给应用研发者，通过灵活的高抽象度降低研发者参与运维工作的认知负担，使其自助完成业务交付运维的研发及操作。
+以 Kubernetes 为代表的云原生技术正成为现代企业及云厂商的一方技术，已形成面向全球的社区生态。Kubernetes API 支持面向终态、数据化（as Data）的定义方式，声明式、版本化、面向资源的可扩展 API，可终止、可预测的服务端模拟验证机制，这些优秀的设计使其天然具有一致的接入方式，完善的资源定义和扩展方式，以及客户端友好的验证、执行机制，原生具备了传统三方 API 接入层技术产品的典型能力。但从另一个角度看，现代应用也将同时长期依赖未云原生化的 PaaS、IaaS 层服务，这使得企业内应用交付运维依赖分散变化的技术和平台，复杂割裂的工作界面和流程。此外，Kubernetes 是一种用于构建平台的平台技术，面向大量的非专家型应用研发者，Kubernetes 自身的技术复杂性和大量面向资源的使用界面制约了在企业内部以开放、民主方式建设更为广泛的 DevOps 生态。最后，平台侧缺少有效手段让持续增长的平台能力简单、一致、可控地开放给应用研发者，通过面向业务的高抽象度降低研发者参与运维工作的认知负担，使其自助完成业务交付运维的研发及操作。
```
Comment (SparkYuan) [inline docs/user_docs/intro/kusion-intro.md:22]: “可终止、可预测的服务端模拟验证机制”，可终止是指的什么？

File: docs/user_docs/intro/kusion-intro.md (hunk 1 above)
Comment (ldxdl) [reply docs/user_docs/intro/kusion-intro.md:30]: replace it with replayable(可重放)

File: docs/user_docs/intro/kusion-intro-en.md (hunk 2 above)
Comment (ldxdl) [reply docs/user_docs/intro/kusion-intro-en.md:7]: fixed.

File: docs/user_docs/intro/kusion-intro-en.md (hunk 3 above)
Comment (ldxdl) [reply docs/user_docs/intro/kusion-intro-en.md:10]: platform stands for all cloud-native and non-cloud-native techs

File: docs/user_docs/intro/kusion-intro.md (hunk 4 above)
Comment (ldxdl) [reply docs/user_docs/intro/kusion-intro.md:22]: request to kube api server must be terminable to enable server-side dry-run without exhausting server resources

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**